tests/arch/x86/cpu_scrubs_regs/src/main.c

```diff
@@ -69,7 +69,7 @@ static inline void z_vrfy_test_cpu_write_reg(void)
  *
  * @ingroup kernel_memprotect_tests
  */
-void test_syscall_cpu_scrubs_regs(void)
+ZTEST_USER(test_x86_cpu_scrubs_regs, test_syscall_cpu_scrubs_regs)
 {
 #if CONFIG_X86
```

Comment:
Note, `testcase.yaml` sets `arch_allow: x86` so this `#if CONFIG_X86` will always be true, can be removed

Reply:
Thanks for pointing this out, fixed, thanks!

Reply:
Got it, I was also considering should I keep test_ or not. Seems like leave it only for testcase is better.